samples/README.md

```diff
@@ -49,24 +49,24 @@ These samples are provided:
 * [azure-deployment](azure-deployment) - Deploys on Microsoft Azure Kubernetes
   Service (AKS).
 * [azure-monitor](azure-monitor) - Enables Azure Monitor to collect metrics
```

Comment:
The SAS Viya Monitoring project deploys components intended to monitor SAS Viya (in all flavors) and not just the SAS Viya platform.  These edits suggest the monitoring is limited to the SAS Viya platform.